README.md

```diff
@@ -15,8 +23,7 @@ AeroGear SDK for Java Script, Apache Cordova and React Native.
 
 AeroGear Services SDK consist of set of separate SDKs
 
-- [Core](./docs/core/README.adoc): Common base for all SDKs
-- [Metrics](./docs/metrics/README.adoc): Mobile application metrics SDK
+- [Core](./docs/core.adoc): Common base for all SDKs
```

Comment:
@wtrocki @paolobueno are we going to have metrics separated in the end? Let's add `Auth` too, no?

Reply:
Metrics is separate docs.

Reply:
@wtrocki We have a PR here covering auth documentation https://github.com/aerogear/aerogear-js-sdk/pull/39

Reply:
Great. Let's split metrics then.